change_log/next/reports_no_scroll_issue.yml

```diff
@@ -0,0 +1 @@
+Bug Fixes: "Fixes AC-3061, where the Balance Sheet and Profit and Loss pages would not scroll when opening the full screen dialog"
```

Comment:
I think it shouldn't include any reference to any ticket (don't forget it's an open source project).

Reply:
@mob1970 @alex-buchan it is ok to reference the Jira ticket number if it helps, that'll not mean anything to anyone without access to Jira. But you shouldn't provide a link to our Jira app, and you shouldn't mentioned anything product related, for example what is broken in the app etc, and you shouldn't include screenshots of the bug in the app. Keep it focussed on the component, and include screenshots from the demo site etc.

Reply:
But its also how your phrase it... saying "This fixes AC-3061" will be confusing to anyone external. But "AC-3061 - This fixes an issue with..." it is just adding a reference to the release note which is fine and won't be confusing.